packages/features/ee/teams/components/TeamsListing.tsx

```diff
@@ -65,31 +64,80 @@ export function TeamsListing() {
           <TeamList teams={invites} />
         </div>
       )}
+
       {isLoading && <SkeletonLoaderTeamList />}
+
+      <UpgradeScreen
+        title="calcom_is_better_with_team"
+        description="add_your_team_members"
+        features={features}
+        background="/team-banner-background.jpg"
+        buttons={
+          <div className="space-y-2 rtl:space-x-reverse sm:space-x-2">
+            <ButtonGroup>
+              <Button color="primary" href={`${WEBAPP_URL}/settings/teams/new`}>
+                {t("create_team")}
+              </Button>
+              <Button color="secondary" href="https://go.cal.com/teams-video" target="_blank">
+                {t("learn_more")}
+              </Button>
+            </ButtonGroup>
+          </div>
+        }>
+        <>
+          <EmptyScreen
+            Icon={Icon.FiUsers}
+            headline={t("no_teams")}
+            description={t("no_teams_description")}
+            buttonRaw={
+              <Button color="secondary" href={`${WEBAPP_URL}/settings/teams/new`}>
+                {t("create_team")}
+              </Button>
+            }
+          />
+          {teams.length > 0 && <TeamList teams={teams} />}
+        </>
+      </UpgradeScreen>
```

Comment:
@sean-brydon @JeroenReumkens I hope to move as much of "team" logic into the upgrade screen so the pages dont need to call teams individually to check if someone has a team or not.

please comment on the props API used here